### PR TITLE
Factorial code to the html file.

### PR DESCRIPTION
**Pull request by McDoxa**
Added the JavaScript code to compute the factorial of a number.

### CODE REVIEW

File: functions.html [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
+            else if (n === 0){ // checks if number is 0 and returns 1 as the answer.
+              let answer = 1;
+              return console.log(`The factorial of ${n} = ${answer}.`);}
+            else {
+              answer = n * factorial(n-1);
+              return console.log(`The factorial of ${n} is ${answer}`);
+              }
+          }
+          // log the result to the console.
+          let userInput = ''; // initializes the number from the user
```
Comment (basir) [inline functions.html]: change userInput to a number 8 instead of '' empty.

Comment (McDoxa) [reply functions.html]: I have effected the change and pushed the commit to the same branch. Do I need to create another pull request?

Comment (basir) [reply functions.html]: Yes please create another pull request so that I can review your code and probably merge it.

Comment (basir) [reply functions.html]: I checked your repo and didn't find any new pushes on it. your last push goes back to 4 days ago. Please push your changes on the branch and then submit new pull request.

Comment (McDoxa) [reply functions.html]: I tried pushing the changes to the same branch (seyidoxa) but not making any headway. So i pushed to a new branch (doxa) and made a new pull request. Thank you

**Pull request merged**